css/03-generic.css

```diff
@@ -131,10 +131,14 @@ input[type="submit"]:hover {
 }
 
 /* de-hyphenate large pullquotes */
-.wp-block-pullquote .alignwide, .wp-block-pullquote .alignfull {
-	hyphens: manual;
+.wp-block-pullquote .alignwide, .wp-block-pullquote .alignwide blockquote, .wp-block-pullquote .alignfull, .wp-block-pullquote .alignfull blockquote {
+     hyphens: manual;
 }
 
+/* wide block separators and hr clear both */
+.entry-content .wp-block-separator.is-style-wide, .entry-content hr.is-style-wide {
+	clear: both;
+
```

Comment:
@slambert Looks like a missing closing curly brace below this clear declaration. Can you post a link to where this applies on the site? Just to confirm that clear is the appropriate solution - it seems like the theme should be taking care of this.

Reply:
You're right about the curly brace.

**I pulled this out of the commit** because I'm not sure I'm doing it the best way and I'm hoping @zoracreates could tackle this better (and probably faster). I made a separate issue for it: https://github.com/thec4aa/c4aa-org/issues/61

This is a good example of where it applies: http://backtobasics.c4aa.org/advising/

When the page is narrow, everything looks good:

<img width="678" alt="Screen Shot 2019-07-30 at 9 04 08 AM" src="https://user-images.githubusercontent.com/67271/62131490-519d6f00-b2a9-11e9-8f97-1ed3e5abae21.png">

If you widen it out:

<img width="1863" alt="Screen Shot 2019-07-30 at 9 03 55 AM" src="https://user-images.githubusercontent.com/67271/62131750-bfe23180-b2a9-11e9-867a-f8762e60a52f.png">